src/moderngl.cpp

```diff
@@ -273,11 +273,12 @@ struct MGLTextureCube {
     int texture_obj;
     int width;
     int height;
-    int depth;
+    bool depth;
```

Comment:
int is preferred for booleans for better C compatibility, mostly due to match the return type of the used Python C Api like the PyObject_IsTrue or the PyArg_ParseTuple.. "p" format.

below released was defined as bool as it is some temporary workaround for a not yet solved problem.